helpers/base.sh

```diff
@@ -196,28 +176,41 @@ check_python_version() {
     fi
 }
 
-# Chooses a suitable compressor depending on which parameters passed (or didn't
+# Execute a suitable compression depending on which parameters passed (or didn't
 # pass) the user.
```

Comment:
Now the function has a side-effect (it modifies `IMAGE`), so it should be mentioned in this docstring.
```
The function has a side-effect -- it modifies `IMAGE` if one of the following environment variables is set to true.
```